content/blog/continuous-delivery-to-any-cloud-using-github-actions-and-pulumi/index.md

```diff
@@ -140,7 +142,7 @@ integration and, more importantly, context added by the Pulumi bot to
 your Pull Requests about what a deployment will do before it's even
 done, as shown above.
 
-Note that this is optional with GitHub Actions, but worthwhile -- it
+Note that this is optional with GitHub Actions, but worthwhile -- it's
```

Comment:
Not introduced in this PR, but it's all over the site. `--` and `---` shouldn't have spaces before and after them. I've started fixing where I can, and it's more a minor nit.